about/views.py

```diff
@@ -55,6 +56,7 @@ def profile(request):
 			person.year 	= profileForm.cleaned_data["year"]
 			person.position	= profileForm.cleaned_data["position"]
 			person.bio 		= profileForm.cleaned_data["bio"]
+			person.link 	= profileForm.cleaned_data["link"]
```

Comment:
I don't see this field being added to ProfileForm anywhere? I'll add it before merging then